GameAnalytics.gd

```diff
@@ -1,4 +1,4 @@
-extends Node
+extends Reference
```

Comment:
This change fixes possible memory leaks since `Node` must be "freed" manually.
This class does not need to be added to the tree, so `Reference` is more appropriate.